src/test/java/gov/healthit/chpl/aqa/api/payLoad/complaintsPayload.java

```diff
@@ -0,0 +1,106 @@
+package gov.healthit.chpl.aqa.api.payLoad;
+
+import gov.healthit.chpl.aqa.api.asserts.ComplaintsAsserts;
+
+public final class ComplaintsPayload {
```

Comment:
A class with the name "ComplaintsPayload" should be in a file named "ComplaintsPayload.java". This class was originally named "complaintsPayload", which doesn't meet our checkstyle rules. This file should be removed

Reply:
I don't see that file on my local anymore, how to fix this?

Reply:
removed it